code/game/objects/items/mail.dm

```diff
@@ -0,0 +1,262 @@
+
+/// Mail is tamper-evident and unresealable, postmarked by CentCom for an individual recepient.
+/obj/item/mail
+	name = "mail"
+	gender = NEUTER
+	desc = "An officially postmarked, tamper-evident parcel regulated by CentCom and made of high-quality materials."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail_small"
+	inhand_icon_state = "paper"
+	worn_icon_state = "paper"
+	item_flags = NOBLUDGEON
+	w_class = WEIGHT_CLASS_SMALL
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
+	/// destination tagging for the mail sorter
+	var/sortTag = 0
+	/// who this mail is for and who can open it
+	var/mob/recipient
```

Comment:
mob living carbon human  not mob

Reply:
Additionally if this mob gets deleted before this mail is opened it will hard delete add protection for that

Reply:
it should store a string copy of their ckey, not the mob